docs/_static/vdr/v2.yaml

```diff
@@ -147,71 +147,6 @@ paths:
                 $ref: '#/components/schemas/Service'
         default:
           $ref: '../common/error_response.yaml'
-  /internal/vdr/v2/did/{did}/service/{serviceId}:
-    parameters:
-      - name: did
-        in: path
-        description: URL encoded DID.
-        required: true
-        content:
-          plain/text:
-            schema:
-              type: string
-              example: "did:web:example.com:iam:123"
-      - name: serviceId
-        in: path
-        description: URL encoded ID identifying the service.
-        required: true
-        example:
-          - "did:web:example.com:iam:013c6fda-73e8-45ee-9220-48652dba854b#3106f751-59e3-440f-b57b-39a96a2da6c6"
-          - "#3106f751-59e3-440f-b57b-39a96a2da6c6"
-        schema:
-          type: string
-    delete:
-      summary: Delete a specific service
-      description: |
-        Removes the service from the DID Document.
-        No cascading will happen for references to the service. 
-
-        error returns:
-        * 404 - Corresponding DID document or verification method could not be found
-        * 500 - An error occurred while processing the request
-      tags:
-        - DID
-      operationId: deleteService
-      responses:
-        "204":
-          description: The service was successfully deleted
-        default:
-          $ref: '../common/error_response.yaml'
-    put:
-      summary: Updates a service in the DID document.
-      description: |
-        It updates the given service in the DID Document.
-
-        error returns:
-        * 400 - Returned in case of malformed DID or service
-        * 404 - Corresponding DID document could not be found
-        * 500 - An error occurred while processing the request
-      tags:
-        - DID
-      operationId: updateService
-      requestBody:
-        description: New version of the service.
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/Service'
-      responses:
-        "200":
-          description: "Service has been updated successfully. Returns the service."
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Service'
-        default:
-          $ref: '../common/error_response.yaml'
     get:
       summary: "Filters services of a resolved DID document"
```

Comment:
The diff doesn't match the PR, so just confirming: you moved this `get ` to a different path?

Reply:
a, weird yes. I moved `FilterServices` to `/service`, it was under `/service/{id}` by accident (which doesn't make sense since it's meant to filter the services, not pre-select a specific one).